src/sumstats.cpp

```diff
@@ -91,7 +91,7 @@ SEXP IVWCpp(
     arma::colvec se_anno = se.elem(key);
     arma::mat d_anno = arma::diagmat(se_anno);
     arma::mat v_anno = d_anno * ld_anno * d_anno;
-    arma::mat v_anno_inv = arma::inv_sympd(v_anno);
+    arma::mat v_anno_inv = arma::pinv(v_anno);
```

Comment:
Is this change strictly related to the eps adjustment? Could you comment on the motivation for swapping this out?

Reply:
This ensures the function behaves as expected if `eps` were set to 0.

Reply:
Ah makes sense. Worth leaving a comment?

Reply:
Sure, just added.